### PR TITLE
404 page: Add error page

### PR DESCRIPTION
**Pull request by pedrojgm**

### CODE REVIEW

File: 404-page.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            canvas.width = innerWidth;
+            canvas.height = innerHeight;
+            const brush = canvas.getContext('2d');
+
+            //funciton to construct circles
+            function Circle(x, y, velocityX, velocityY, radius) {
+                this.x = x;
+                this.y = y;
+                this.velocityX = velocityX;
+                this.velocityY = velocityY;
```
Comment (gonzalo-plaza) [inline 404-page.html:59]: @pedrojgm I think that the name of the parameters can be more descriptive.

**Comment by gonzalo-plaza** [comment]
@pedrojgm Please, change the name of PR, normaly when you create the PR, GITHUB apply the name of the commit, and is no recomended use the name of the commit in the name of PR, for example, "Create 404 page" can be a good name. And please, fix the problem with the test. If need help, I am here,.

**Comment by gonzalo-plaza** [comment]
@pedrojgm I have review the JS and I have understand that you are doing, and it is okay, but before you must fix the reviews that I have doing, because I think that can broke the funcionality of the circles. First, fix the reviews and after contact me.

File: 404-page.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            z-index: 99;
+            position: fixed;
+        }
+
+    </style>
+
+    <body>
+        <canvas id="canvas"></canvas>
+        <div class="first-block">
+            <h1>404</h1>
```
[the remark this answers is not in the document]
Comment (JuanJosePG) [reply 404-page.html:42]: Also, I saw that you use h1 in styles to modify the title, this is a bad practice. Firstly, set up a class for the title and then, replace it.

[the remark this answers is not in the document]
Comment (JuanJosePG) [reply 404-page.html:42]: For the subtitle, don't use h3 tag, use h2. To improve the SEO.

File: 404-page.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <meta charset="utf-8" />
+
+    <head>
+        <title>404 - Open Slot Studios</title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
```
[the remark this answers is not in the document]
Comment (pedrojgm) [reply 404-page.html:9]: Hi, we resolve this in the last meeting. Ty for your comment.

**Comment by gonzalo-plaza** [comment]
@pedrojgm for me this is okay to can be merged. Please @JuanJosePG review this to can merge.

File: 404-page.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        }
+
+        .scene {
+            z-index: 99;
+            position: fixed;
+        }
+    </style>
+
+    <body>
+        <canvas id="canvas" class="scene"></canvas>
```
Comment (Andresws12) [inline 404-page.html:38]: @pedrojgm Maybe, scene is not sufficiently descriptive to understand what is the functionality from this canvas and his css

File: 404-page.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+        <div class="container">
```
Comment (Andresws12) [inline 404-page.html:39]: @pedrojgm Container from? I would say container-error-page for example to be more descriptive

**Comment by gonzalo-plaza** [comment]
@pedrojgm Remember metion the person that have review your code in a comment to notify that you have fix the error. @Andresws12

**Pull request closed without merging**